iml-action-client/Cargo.toml

```diff
@@ -5,11 +5,15 @@ authors = ["IML Team <iml@whamcloud.com>"]
 edition = "2018"
 
 [dependencies]
+bytes = "0.5"
 futures = "0.3"
 iml-action-runner = { path = "../iml-services/iml-action-runner", version = "0.3" }
 iml-manager-env = { path = "../iml-manager-env", version = "0.3" }
 iml-wire-types = { path = "../iml-wire-types", version = "0.3" }
-reqwest = { version = "0.10", features = ["default-tls", "native-tls", "json", "stream"] }
+hyper = { version = "0.13", default-features = false, features = ["tcp"] }
+# this can be used once hyperlocal#36 lands
+#hyperlocal = { version = "0.7", features = [ "client" ], default_features = false }
+hyperlocal = { version = "0.7" }
```

Comment:
If this is enabled can we remove the comments above?

Reply:
we only need the "client" features of hyperlocal, so I'd like to keep our footprint small.  I'm still waiting on https://github.com/softprops/hyperlocal/pull/36 to land